neuro_san/internals/messages/agent_framework_message.py

```diff
@@ -28,6 +28,7 @@ class AgentFrameworkMessage(BaseMessage):
     def __init__(self, content: Union[str, List[Union[str, Dict]]] = None,
                  chat_context: Dict[str, Any] = None,
                  sly_data: Dict[str, Any] = None,
+                 structure: Dict[str, Any] = None,
```

Comment:
Allow for adding a structure to the AgentFrameworkMessage.
Not used just yet, but will eventually.